src/patternfly/utilities/Sizing/sizing.scss

```diff
@@ -43,4 +44,62 @@ $pf-u-height-options: (
 @include pf-utility-builder($pf-u-width-options, $pf-global--breakpoint-list);
 @include pf-utility-builder($pf-u-height-options, $pf-global--breakpoint-list);
 
+.pf-u-min-width,
+.pf-u-max-width {
+  width: 100% !important;
+}
+
+.pf-u-min-width {
+  --pf-u-min-width--MinWidth: 0;
+
+  @include pf-build-css-variable-stack(
+    "min-width",
+    "--pf-u-min-width--MinWidth",
+    $pf-u-sizing--min-max--breakpoint-map,
+    $important: true
+  );
+}
+
+.pf-u-max-width {
+  --pf-u-max-width--MaxWidth: auto;
+
+  @include pf-build-css-variable-stack(
+    "max-width",
+    "--pf-u-max-width--MaxWidth",
+    $pf-u-sizing--min-max--breakpoint-map,
+    $important: true
+  );
+
+  overflow: auto !important;
```

Comment:
I'm not sure if I would expect overflow/triggering a scrollbar to be set by default or not. The other sizing classes don't set overflow, and from the utility name, I would expect it to just set that property. Were you thinking it would be best if it did? Scrollable/overflow could also potentially be a utility.

Reply:
opened [4147](https://github.com/patternfly/patternfly/issues/4147)